.gitpod.yml

```diff
@@ -8,7 +8,7 @@ ports:
 tasks:
   - name: connect
     command: |
-      python3 $GITPOD_REPO_ROOT/generate_key.py
+      echo "${DBT_SNOWFLAKE_PRIVATE_KEY}" | sed -e "s/-----BEGIN PRIVATE KEY-----/&\n/" -e "s/-----END PRIVATE KEY-----/\n&/" -e "s/\S\{64\}/&\n/g" > $GITPOD_REPO_ROOT/profiles/private_key.p8
```

Comment:
@YanicM i prefer this solution over the previous python script - it's not overly complex and happens in front of the user! still, a short comment to explain what's happening here would be very helpful imo!

Reply:
done